src/aequitas/plotting.py

```diff
@@ -0,0 +1,1064 @@
+
+        :return: Integer indicating relative index of reference group value row.
+        """
+        # get absolute metric name from passed group metric (vs. a disparity name)
+        abs_metric = "".join(group_metric.split('_disparity'))
+
+        all_ref_groups = self.__assemble_ref_groups(disparities_table, ref_group_flag)
+        ref_group_name = all_ref_groups[attribute_name][abs_metric]
+
+        # get index for row associated with reference group for that model
+        ind = list(disparities_table.loc[(disparities_table['attribute_name'] == attribute_name) &
+                                     (disparities_table['attribute_value'] == ref_group_name) &
+                                     (disparities_table['model_id'] == model_id)].index)
+
+        # there should only ever be one item in list, but JIC, select first
+        idx = ind[0]
+
+        relative_ind = disparities_table.index.get_loc(idx)
+        return relative_ind, ref_group_name
+
+
+    def plot_group_metric(self, group_table, group_metric, ax=None, ax_lim=None,
+                          title=True, label_dict=None,
+                          min_group_size = None):
+        '''
+        Plot a single group metric's absolute metrics
+
+        :param group_table: A group table
+        :param group_metric: The metric to plot. Must be a column in the group_table
+        :param ax: A matplotlib Axis. If not passed a new figure will be created.
+        :param ax_lim: Maximum value on x-axis, used to match axes across subplots
+            when plotting multiple metrics. Default is None.
+        :param title: Whether a title should be added to the plot. Default is True.
+        :param label_dict: Optional dictionary of replacement labels for data.
+            Default is None.
+        :param min_group_size: Minimum size for groups to include in visualization
+            (as a proportion of total sample)
+
+        :return: matplotlib.Axis
+        '''
+        assert (group_metric in group_table.columns), \
+            f"Specified disparity metric '{group_metric}' not in 'group_table'."
+
+        if any(group_table[group_metric].isnull()):
+            raise IOError(f"Cannot plot {group_metric}, has NaN values.")
+
+        if ax is None:
+            fig, ax = plt.subplots(figsize=(10, 5))
+
+        height_of_bar = 1
+        attribute_names = group_table.attribute_name.unique()
+        tick_indices = []
+        next_bar_height = 0
+
+        if min_group_size:
+            if min_group_size > (group_table.group_size.max() / group_table.group_size.sum()):
+                raise Exception(f"'min_group_size' proportion specified: '{min_group_size}' "
+                                f"is larger than all groups in sample.")
+
+            min_size = min_group_size * group_table.group_size.sum()
+            group_table = group_table.loc[group_table['group_size'] >= min_size]
+
+        label_position_values = list(group_table[group_metric].values)
```

Comment:
might not matter much for this, but FYI, for the operations you're doing -- `.pop(0)` -- a `collections.deque` might make more sense [[docs]](https://docs.python.org/3.7/library/collections.html#collections.deque)